docs/v2_linter.md

```diff
@@ -78,3 +72,33 @@ all the checks of the linter, it can be skipped from `pylint` adding the followi
 from conans import ConanFile, CMake, tools
 ...
 ```
+
+---
+
+## Running the linter locally
+
+It is possible to run the linter locally the same way it is being run [using Github actions](../.github/workflows/linter-conan-v2.yml):
+
+ * (Recommended) Use a dedicated Python virtualenv.
+ * Ensure you have required tools installed: `conan` and `pylint` (better to uses fixed versions)
+
+   ```
+   pip install conan~=1.0 pylint==2.14
+   ```
+
+ * Set environment variable `PYTHONPATH` to the root of the repository
+
+   ```
+   export PYTHONPATH=your/path/conan-center-index
+   ```
+
+  * Now you just need to execute the `pylint` commands:
+
+    ```
+    # Lint a recipe:
+    pylint --rcfile=linter/pylintrc_recipe recipes/boost/all/conanfile.py
```

Comment:
I was interested in this topic so I tried this locally and have the following result:

```
$ pylint --rcfile=linter/pylintrc_recipe recipes/boost/all/conanfile.py
************* Module linter/pylintrc_recipe
linter/pylintrc_recipe:1:0: W0012: Unknown option value for '--enable', expected a valid pylint message and got 'conan-bad-name' (unknown-option-value)
linter/pylintrc_recipe:1:0: W0012: Unknown option value for '--enable', expected a valid pylint message and got 'conan-missing-name' (unknown-option-value)
linter/pylintrc_recipe:1:0: W0012: Unknown option value for '--enable', expected a valid pylint message and got 'conan-import-conanfile' (unknown-option-value)
************* Module Command line or configuration file
Command line or configuration file:1:0: E0013: Plugin 'linter.transform_imports' is impossible to load, is it installed ? ('No module named 'linter'') (bad-plugin-value)
Command line or configuration file:1:0: E0013: Plugin 'linter.transform_conanfile' is impossible to load, is it installed ? ('No module named 'linter'') (bad-plugin-value)
Command line or configuration file:1:0: E0013: Plugin 'linter.conanv2_transition' is impossible to load, is it installed ? ('No module named 'linter'') (bad-plugin-value)
************* Module conanfile
recipes/boost/all/conanfile.py:1:0: C0302: Too many lines in module (1734/1000) (too-many-lines)
recipes/boost/all/conanfile.py:157:21: E1101: Instance of 'BoostConan' has no 'conan_data' member (no-member)
recipes/boost/all/conanfile.py:158:12: E1101: Instance of 'BoostConan' has no 'copy' member (no-member)
recipes/boost/all/conanfile.py:161:8: E1101: Instance of 'BoostConan' has no 'copy' member (no-member)
recipes/boost/all/conanfile.py:191:49: E1101: Instance of 'BoostConan' has no 'recipe_folder' member (no-member)
recipes/boost/all/conanfile.py:257:4: R0912: Too many branches (28/12) (too-many-branches)
recipes/boost/all/conanfile.py:257:4: R0915: Too many statements (63/50) (too-many-statements)
recipes/boost/all/conanfile.py:368:4: R0912: Too many branches (14/12) (too-many-branches)
recipes/boost/all/conanfile.py:409:4: R0912: Too many branches (37/12) (too-many-branches)
recipes/boost/all/conanfile.py:409:4: R0915: Too many statements (56/50) (too-many-statements)
recipes/boost/all/conanfile.py:548:12: E1101: Instance of 'BoostConan' has no 'info' member (no-member)
recipes/boost/all/conanfile.py:551:12: E1101: Instance of 'BoostConan' has no 'info' member (no-member)
recipes/boost/all/conanfile.py:552:12: E1101: Instance of 'BoostConan' has no 'info' member (no-member)
recipes/boost/all/conanfile.py:554:16: E1101: Instance of 'BoostConan' has no 'info' member (no-member)
recipes/boost/all/conanfile.py:555:16: E1101: Instance of 'BoostConan' has no 'info' member (no-member)
recipes/boost/all/conanfile.py:556:16: E1101: Instance of 'BoostConan' has no 'info' member (no-member)
recipes/boost/all/conanfile.py:557:16: E1101: Instance of 'BoostConan' has no 'info' member (no-member)
recipes/boost/all/conanfile.py:559:20: E1101: Instance of 'BoostConan' has no 'info' member (no-member)
recipes/boost/all/conanfile.py:561:16: E1101: Instance of 'BoostConan' has no 'info' member (no-member)
recipes/boost/all/conanfile.py:565:12: E1101: Instance of 'BoostConan' has no 'build_requires' member (no-member)
recipes/boost/all/conanfile.py:568:20: E1101: Instance of 'BoostConan' has no 'conan_data' member (no-member)
recipes/boost/all/conanfile.py:636:31: E1101: Instance of 'BoostConan' has no 'info' member (no-member)
recipes/boost/all/conanfile.py:708:4: R0914: Too many local variables (20/15) (too-many-locals)
recipes/boost/all/conanfile.py:796:27: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
recipes/boost/all/conanfile.py:916:4: R0911: Too many return statements (8/6) (too-many-return-statements)
recipes/boost/all/conanfile.py:1145:12: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
recipes/boost/all/conanfile.py:963:4: R0912: Too many branches (50/12) (too-many-branches)
recipes/boost/all/conanfile.py:963:4: R0915: Too many statements (128/50) (too-many-statements)
recipes/boost/all/conanfile.py:1204:4: R0912: Too many branches (20/12) (too-many-branches)
recipes/boost/all/conanfile.py:1204:4: R0915: Too many statements (61/50) (too-many-statements)
recipes/boost/all/conanfile.py:1297:4: R0911: Too many return statements (10/6) (too-many-return-statements)
recipes/boost/all/conanfile.py:1356:8: E1101: Instance of 'BoostConan' has no 'copy' member (no-member)
recipes/boost/all/conanfile.py:1360:12: E1101: Instance of 'BoostConan' has no 'copy' member (no-member)
recipes/boost/all/conanfile.py:1410:4: R0914: Too many local variables (26/15) (too-many-locals)
recipes/boost/all/conanfile.py:1600:64: C0121: Comparison 'self.options.get_safe('with_stacktrace_backtrace') == False' should be 'self.options.get_safe('with_stacktrace_backtrace') is False' if checking for the singleton value False, or 'not self.options.get_safe('with_stacktrace_backtrace')' if testing for falsiness (singleton-comparison)
recipes/boost/all/conanfile.py:1646:23: C0121: Comparison 'self.options.get_safe(requirement, None) == False' should be 'self.options.get_safe(requirement, None) is False' if checking for the singleton value False, or 'not self.options.get_safe(requirement, None)' if testing for falsiness (singleton-comparison)
recipes/boost/all/conanfile.py:1410:4: R0912: Too many branches (59/12) (too-many-branches)
recipes/boost/all/conanfile.py:1410:4: R0915: Too many statements (204/50) (too-many-statements)
recipes/boost/all/conanfile.py:9:0: C0412: Imports from package conan are not grouped (ungrouped-imports)
```

Are these errors normal ?

Reply:
Did you set the `PYTHONPATH` env variable? 

Errors like the following are telling that `linter.transform_imports` was not found

```
Command line or configuration file:1:0: E0013: Plugin 'linter.transform_imports' is impossible to load, is it installed ? ('No module named 'linter'') (bad-plugin-value)
```

Reply:
👍  

Please, feel free to suggest better wording or anything you think that can improve the instructions.